### PR TITLE
Ruowan/postman collection

### PR DESCRIPTION
**Pull request by ruowan**

### CODE REVIEW

File: lib/testScenario/postmanCollectionRunnerClient.ts [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+    }
+  }
+
+  private addAsLongRunningOperationItem(item: Item) {
+    this.collectionEnv.set(`${item.name}_polling_url`, "<polling_url>", "string");
+    const longRunningEvent = new Event({
+      listen: "test",
+      script: {
+        type: "text/javascript",
+        exec: `pm.environment.set("${item.name}_polling_url", pm.response.headers.get('Location')||pm.response.headers.get('Azure-AsyncOperation')||"https://postman-echo.com/delay/10")`,
```
Comment (PhoenixHe-msft) [inline lib/testScenario/postmanCollectionRunnerClient.ts:194]: Header may be lower cased. And you need to support Operation-Location

File: lib/testScenario/postmanCollectionRunnerClient.ts [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+      listen: "prerequest",
+      script: {
+        type: "text/javascript",
+        exec: `postman.setNextRequest('${nextRequestName}')`,
+      },
+    });
+    ret.events.add(event);
+    return ret;
+  }
+  public aadAuthAccessTokenItem(env: VariableEnv): Item {
```
Comment (PhoenixHe-msft) [inline lib/testScenario/postmanCollectionRunnerClient.ts:395]: Format

File: lib/testScenario/postmanCollectionRunnerClient.ts [hunk 3]
```diff
@@ -0,0 +1,437 @@
...
+      name: "get Azure AAD Token",
+    } as ItemDefinition);
+    ret.request = new Request({
+      method: "post",
+      url: "",
+      body: {
+        mode: "urlencoded",
+        urlencoded: [
+          { key: "grant_type", value: "client_credentials" },
+          { key: "client_id", value: "{{client_id}}" },
```
Comment (PhoenixHe-msft) [inline lib/testScenario/postmanCollectionRunnerClient.ts:407]: User must login with service principal?

File: lib/testScenario/postmanCollectionRunnerClient.ts (hunk 1 above)
Comment (ruowan) [reply lib/testScenario/postmanCollectionRunnerClient.ts:214]: What's Operation-Location?

**Pull request merged**